SPECS/hdf5/hdf5.spec

```diff
@@ -1,6 +1,7 @@
 %global macrosdir %(d=%{_rpmconfigdir}/macros.d; [ -d $d ] || d=%{_sysconfdir}/rpm; echo $d)
 %define version_main %(echo %{version} | cut -d. -f-2)
-%global so_version 200
+%global so_version_1 310
+%global so_version_2 311
```

Comment:
This change will require a `Release` bump in all specs (`python-h5py`, `gdal`, and `netcdf`) with `BuildRequires: hdf5-devel`. This will cause them to re-build with the updated package and thus depend on the updated libraries. Otherwise, installing the current version of these other packages will prevent usage of the new version of `hdf5` due to their run-time dependency on the 200 version of the library.

Reply:
I've made the change to `python-h5py`.
I'm not sure what `gdal` or `netcdf` are. I could not find packages with those names.

Reply:
fixed gdal and netcdf under specs-extended.